Sources/SnapshotPreferences/RenderingModePreference.swift

```diff
@@ -18,11 +18,36 @@ struct RenderingModePreferenceKey: PreferenceKey {
 }
 
 extension View {
-  @available(macOS, unavailable)
-  @available(watchOS, unavailable)
-  @available(visionOS, unavailable)
-  @available(tvOS, unavailable)
-  public func emergeRenderingMode(_ renderingMode: EmergeRenderingMode?) -> some View {
-    preference(key: RenderingModePreferenceKey.self, value: renderingMode?.rawValue)
-  }
+    /// Sets the emerge rendering mode for the view.
+    ///
+    /// Use this method to control how the view is rendered for snapshots. You can indicate whether
+    /// to use `.coreAnimation` which will use the CALayer from Quartz or `.uiView` which will use
+    /// UIKit's `drawViewHierarchyInRect` under the hood.
+    ///
+    /// - Note: This method is only available on iOS. It is unavailable on macOS, watchOS, visionOS, and tvOS.
+    ///
+    /// - Parameter renderingMode: An `EmergeRenderingMode` value that specifies the
+    ///   desired rendering mode for emerge transitions. If `nil`, the default rendering
```

Comment:
"emerge transitions"? Not sure what that is, maybe just "the desired rending mode when the view is rendered for snapshots" -> like you say in the earlier paragraph in this comment